### ISSUE TITLE
Problem opening the domain to the external network

### ISSUE THREAD

**Issue by mikropsoft** [getumbrel/umbrel#1832]
I get an error as shown in the screenshot, even though I have opened an external network connection via Keenetic, everything I have done is correct.

![Ekran Görüntüsü (10)](https://github.com/getumbrel/umbrel/assets/75412448/b260e684-1942-4fe4-936a-08043839ef8c)

![Ekran Görüntüsü (11)](https://github.com/getumbrel/umbrel/assets/75412448/d84c06f4-743a-4ae7-93c9-090d96e0d72e)

**Comment by lil5** [comment]
Please don't do this, port 80 will be sending unencrypted packets to your server making a ManInTheMiddle attack very easy (intercepting your login password/token).

Umbrel dev seem not interested in adding 443 https support.

If you want to access Umbrel outside of LAN I suggest using tailscale, this still opens up attacks when not connected to tailscale.

TLDR; wait until umbrel get their act together and add https support.

**Comment by mikropsoft** [comment]
> Please don't do this, port 80 will be sending unencrypted packets to your server making a ManInTheMiddle attack very easy (intercepting your login password/token).
> 
> Umbrel dev seem not interested in adding 443 https support.
> 
> If you want to access Umbrel outside of LAN I suggest using tailscale, this still opens up attacks when not connected to tailscale.
> 
> TLDR; wait until umbrel get their act together and add https support.

Security is not important for me. Also in this setting that appears in keenetic, umbrel works on port 80 so I enter that. When I open it to the external network with keendns I can connect as https, you can investigate that. I was able to connect through Casaos without any problem, I hope they can solve this problem, thanks!
